File: Eclipse-Workspace/PLT_Exercises/src/SymmetricMatrix.java
import java.util.Scanner;

public class SymmetricMatrix {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter The NxN Number : ");
		int N = sc.nextInt();
		int[][] matrix = new int[N][N];
		boolean flag = true;
		for (int i = 0; i < N; i++) {
			for (int j = 0; j < N; j++) {
				System.out.print("Enter The Element [" + i + "]" + "[" + j + "] : ");
				matrix[i][j] = sc.nextInt();
			}
		}
		for (int i = 0; i < N; i++) {
			for (int j = 0; j < N; j++) {
				if (matrix[i][j] != matrix[j][i])
					flag = false;
			}
		}
		if (flag)
			System.out.println("It Is A Symmetric Matrix");
		else
			System.out.println("It Is Not A Symmetric Matrix");

	}

}

File: Training-Workspace/Web Development/Day_Four_Tasks/symmetricMatrix.js
const symmetricMatrix = (matrix) => {
  let flag = true;
  for (let i = 0; i < 3; i++) {
    for (let j = 0; j < 3; j++) {
      if (matrix[i][j] != matrix[j][i]) flag = false;
    }
  }
  if (flag) console.log("It Is A Symmetric Matrix");
  else console.log("It Is Not A Symmetric Matrix");
};

let matrixOne = [
  [1, 2, 3],
  [2, 4, 5],
  [3, 5, 8],
];
let matrixTwo = [
  [1, 2, 4],
  [2, 4, 5],
  [3, 5, 8],
];
symmetricMatrix(matrixOne);
symmetricMatrix(matrixTwo);
